package.json

```diff
@@ -52,6 +52,7 @@
   "dependencies": {
     "@amcharts/amcharts4": "^4.9.12",
     "@apollo/client": "^3.1.5",
+    "@cowprotocol/cow-sdk": "0.0.8-RC.0",
```

Comment:
should we fix the version?

Reply:
I've published as a test, since 0.0.7 was already taken.
You could bump to 0.0.8 and release it again or wait until the sub graph changes are in to make a new release.